CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Contributing
+
+Everyone is welcome to contribute to GoDaddy's Open Source Software.
+Contributing doesn’t just mean submitting pull requests. To get involved,
+you can report or triage bugs, and participate in discussions on the
```

Comment:
Phrasing / flow suggestion: "Contributing doesn't just mean submitting pull requests. You can also get involved by reporting/triaging bugs, or participating in discussions on the evolution of each project."